sos/report/plugins/ssh.py

```diff
@@ -49,7 +51,12 @@ def included_configs(self, sshcfgs):
         """ Include subconfig files """
         # Read configs for any includes and copy those
         try:
-            for sshcfg in sshcfgs:
+            cfgfiles = [
+                f for files in [
+                    glob(copyspec, recursive=True) for copyspec in sshcfgs
+                ] for f in files
+            ]
```

Comment:
I think this could be cleaned up slightly, but it does track as it is.

LGTM either way.